app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list):
+    return sum([teammate.get_rating() for teammate in team])
```

Comment:
It is not necessary write here a list generator. If u remove it code will still work ;)

Reply:
Yes, it would be better to remove square brackets